rousseau_vote/lib/src/widgets/user/badge_slider.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:rousseau_vote/src/l10n/rousseau_localizations.dart';
+import 'package:rousseau_vote/src/models/badge.dart';
+import 'package:rousseau_vote/src/config/app_constants.dart';
+
+class BadgeSlider extends StatelessWidget {
+  const BadgeSlider({this.badges});
+
+  final List<Badge> badges;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 81,
+      child: Row(
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.only(right: 4),
+            child: Text.rich(
+             TextSpan(
+                text: badges.length.toString(),
+                style: TextStyle(
+                  fontSize: 36,
+                  fontWeight: FontWeight.w600,
+                  color: Colors.black,
+                ),
+                children: <TextSpan>[
+                  TextSpan(
+                    text: '\nMeriti:'.toUpperCase(),
```

Comment:
Never hardcode strings! This should be in `it.json` and should be retrieved with `RousseauLocalizations.getText(context, 'my_key')`